api/src/settings/settings.py

```diff
@@ -25,6 +25,9 @@
 insecure_key = "insecure"
 SECRET_KEY = os.getenv("SECRET_KEY", insecure_key)
```

Comment:
Laten we de default `insecure` als secret key eruit slopen en de environment variable `SECRET_KEY` verplicht maken. Je kan hem dan in de docker-compose als `insecure` of `dev` instellen